src/css/base/typography.scss

```diff
@@ -7,6 +7,10 @@ h1, h2, h3, h4, h5, h6 {
   & + p {
     margin-top: 0.5rem;
   }
+
+  & + .push {
```

Comment:
I see only the landing page is using this for contact. 

hmm. usually when i see push / pull classes I see it as `push-X` x being a number of some sorts. just want to know how much to expect the push in the name.